app/views/results/common/_annotations.js.erb

```diff
@@ -298,9 +298,56 @@
     }
   }
 
+  function update_annotation_completion(textArea) {
+    const annotationTextList = document.getElementById('annotation_text_list');
+    const value = textArea[0].value;
+
+    $('#annotation_text_id').val('');
+
+    if (value.length < 2) {
+      $('#annotation_completion').hide();
+      return;
+    }
+
+    $.ajax({
+      method: "get",
+      url: Routes.find_annotation_text_assignment_annotation_categories_url(<%= assignment_id %>),
+      dataType: "json",
+      data: {
+        string: value,
+        assignment_id: <%= assignment_id %>,
```

Comment:
Hmm, I'm not sure you need to include this if the assignment id is passed to the `Routes` url function above.